app/graph/types/query_type.rb

```diff
@@ -115,18 +115,15 @@ def find_public_team(slug:)
   field :project_media,
         ProjectMediaType,
         description:
-          'Information about a project media, The argument should be given like this: "project_media_id,project_id,team_id"',
+          'Information about a project media, The argument should be given like this: "project_media_id,team_id"',
         null: true do
     argument :ids, GraphQL::Types::String, required: true
   end
 
   def project_media(ids:)
-    objid, pid, tid = ids.split(",").map(&:to_i)
+    objid, tid = ids.split(",").map(&:to_i)
     tid = (Team.current.blank? && tid.nil?) ? 0 : (tid || Team.current.id)
-    project = Project.where(id: pid, team_id: tid).last
-    pid = project.nil? ? 0 : project.id
-    Project.current = project
-    objid = ProjectMedia.belonged_to_project(objid, pid, tid) || 0
+    objid = ProjectMedia.belonged_to_team(objid, tid) || 0
```

Comment:
@melsawy , this requires changes to Check Web and other GraphQL clients, no? I remember that in different places of Check Web and other Check API clients we used to call like `1,,2`, `1,null,2`, `1,0,2`, etc. As I suggested before, for changes like this, we may need to keep some temporary code in the GraphQL layer with some deprecation warning.

Reply:
Ah, something like:

```ruby
all_ids = ids.split(",").map(&:to_i)
objid = all_ids.first
tid = all_ids.last
```

Right? Yeah, that's a good idea!